tool/debian/build.sh

```diff
@@ -80,32 +80,25 @@ if [ $BUILD = "ON" ]; then
 fi
 
 # create the folder for the package
-FOLDER_PREFIX="singa"
-if [ -n "$BUILD_ID" ]; then
-  FOLDER_PREFIX=$BUILD_ID/$COMMIT/$OS_VERSION
-fi
+if [ $PYTHON = "ON" ]
+then
+  FOLDER_PREFIX="python-singa"
 
-FOLDER=$FOLDER_PREFIX-cpp
-if [ $CUDA = "ON" ]; then
-  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
+else
+  FOLDER_PREFIX="singa"
 fi
 
-if [ $PYTHON = "ON" ]
+if [ $CUDA = "ON" ]
 then
-  FOLDER=$FOLDER/python-singa
+  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
 else
-  FOLDER=$FOLDER/singa
+  FOLDER=singa-cpp/$FOLDER_PREFIX
 fi
 
 echo "Path: " build/debian/$FOLDER
 mkdir -p build/debian/$FOLDER
```

Comment:
yes. I think we just need the first few lines and then call cmake and make pakcage.
please also  try to reformat the commit message (title).